src/terminal/adapter/InteractDispatch.cpp

```diff
@@ -150,15 +150,10 @@ bool InteractDispatch::MoveCursor(const VTInt row, const VTInt col)
 
     const auto coordCursorShort = til::unwrap_coord(coordCursor);
 
-    // MSFT: 15813316 - Try to use this MoveCursor call to inherit the cursor position.
-    auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
-    RETURN_IF_FAILED(gci.GetVtIo()->SetCursorPosition(coordCursorShort));
-
     // Finally, attempt to set the adjusted cursor position back into the console.
-    auto& cursor = _api.GetTextBuffer().GetCursor();
-    cursor.SetPosition(coordCursorShort);
-    cursor.SetHasMoved(true);
-    return true;
+    const auto api = gsl::not_null{ ServiceLocator::LocateGlobals().api };

```

Comment:
Yeah. I was just trying to get the audit build to pass. But I was thinking that it might have been preferable to put this on the api field in the `Globals` class, because once we start auditing more of the code base, we'll assumedly get the C26429 error everywhere the api is used.